sagify/sagemaker/sagemaker.py

```diff
@@ -158,7 +158,7 @@ def _construct_image_location(self, image_name):
         account = self.boto_session.client('sts').get_caller_identity()['Account']
         region = self.boto_session.region_name
 
-        return '{account}.dkr.ecr.{region}.amazonaws.com/{image}:latest'.format(
+        return '{account}.dkr.ecr.{region}.amazonaws.com/{image}'.format(
```

Comment:
The `docker_tag` work earlier this year sets the _latest_ docker tag by default unless otherwise specified by the user i.e. there is no need to hardcode it here. This was actually causing image names of the `image-name:some-tag:latest` form to be used, that `Docker` and `SageMaker` declined.